### PR TITLE
Bybit - postonly fix

### PR DESCRIPTION
**Pull request by ttodua**
compared to upcoming unified approach https://github.com/ccxt/ccxt/pull/17312 , this PR fixes  postonly & tif "in-place" for bybit

### CODE REVIEW

File: ts/src/bybit.ts [hunk 1]
```diff
@@ -3790,9 +3790,8 @@ export default class bybit extends Exchange {
         if (isLimit) {
             request['price'] = this.priceToPrecision (symbol, price);
         }
-        const exchangeSpecificParam = this.safeString (params, 'time_in_force');
         const timeInForce = this.safeStringLower (params, 'timeInForce');
-        const postOnly = this.isPostOnly (isMarket, exchangeSpecificParam === 'PostOnly', params);
+        const postOnly = this.isPostOnly (isMarket, timeInForce === 'PostOnly', params);
```
Comment (ttodua) [inline ts/src/bybit.ts:3794]: exchange specific key is `timeInForce` as is, not `time_in_force`

File: ts/src/bybit.ts [hunk 2]
```diff
@@ -3790,9 +3791,8 @@ export default class bybit extends Exchange {
...
```
Comment (carlosmiei) [inline ts/src/bybit.ts:3795]: @ttodua did you also check `createSpotOrder`, `createUnifiedMargin` and `createUSDCOrder` ?

Comment (carlosmiei) [reply ts/src/bybit.ts:3795]: @ttodua  ⬆️

Comment (ttodua) [reply ts/src/bybit.ts:3795]: sorry forgot to handle them, switching to it now.

Comment (ttodua) [reply ts/src/bybit.ts]: completed now.

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**